src/x86/adlink-ipi.c

```diff
@@ -113,51 +115,51 @@ static mraa_result_t pwm_period_replace(mraa_pwm_context dev, int period)
 
 static float pwm_read_replace(mraa_pwm_context dev)
 {
-	unsigned char rx_tx_buf[3] = {0};
-
-	if(dev->pin < 9)
-	{
-		if(_fd == -1)
-		{
-			return 0;
-		}
-
-		rx_tx_buf[0] = regIon[dev->pin];
-		if(write(_fd, &(rx_tx_buf[0]), 1) == 1)
-		{
-			if(read(_fd, &(rx_tx_buf[1]), 1) == 1)
-			{
-				return ((rx_tx_buf[1] / 2.55)* 2000);
-			}
-		}	
-	}
-	return 0;
+        unsigned char rx_tx_buf[3] = {0};
+
+        if(dev->pin < 9)
+        {
+                if(_fd == -1)
+                {
+                        return 0;
+                }
+
+                rx_tx_buf[0] = regIon[dev->pin];
+                if(write(_fd, &(rx_tx_buf[0]), 1) == 1)
+                {
+                        if(read(_fd, &(rx_tx_buf[1]), 1) == 1)
+                        {
+                                return (rx_tx_buf[1] / 2.55);
+                        }
+                }
+        }
+        return 0;
 }
```

Comment:
please dont mix whitespace usage, tabs/spaces.

Reply:
Okay @tingleby , I was updated the code(23144de84c00995e5455e0ffc412638e252f4b0e). whether should I pull the request again?